android/src/app/organicmaps/widget/placepage/PlacePageController.java

```diff
@@ -84,7 +84,8 @@ class DefaultBottomSheetCallback extends BottomSheetBehavior.BottomSheetCallback
     public void onStateChanged(@NonNull View bottomSheet, int newState)
     {
       Logger.d(TAG, "State change, new = " + PlacePageUtils.toString(newState));
-      if (PlacePageUtils.isSettlingState(newState) || PlacePageUtils.isDraggingState(newState))
+      if (PlacePageUtils.isSettlingState(newState) || PlacePageUtils.isDraggingState(newState) ||
+          mMwmActivity.isFinishing() || mMwmActivity.isDestroyed())
```

Comment:
Will it help to debug/fix a bug of unnecessary calling `onStateChanged()` if we log these isFinishing and isDestroyed states?

Reply:
E.g. to crash/assert in debug.

Reply:
Isn't onStateChanged a BottomSheet callback that is called on any state change, not only after onSaveInstanceState?